build.gradle

```diff
@@ -424,8 +424,10 @@ configurations {
             force "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:${versions.jackson}"
             force "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
             force "io.netty:netty-buffer:${versions.netty}"
+            force "io.netty:netty-codec:${versions.netty}"
             force "io.netty:netty-common:${versions.netty}"
             force "io.netty:netty-handler:${versions.netty}"
+            force "io.netty:netty-resolver:${versions.netty}"
```

Comment:
I believe we can skip this change.

Reply:
ok, I will remove. I was facing issues with dependency resolution and added this to get beyond the issue.